src/service.rs

```diff
@@ -628,12 +639,6 @@ impl Service {
 
                 self.send_event(Discv5Event::TalkRequest(req));
             }
-            RequestBody::RegisterTopic { .. } => {
-                debug!("Received RegisterTopic request which is unimplemented");
-            }
-            RequestBody::TopicQuery { .. } => {
-                debug!("Received TopicQuery request which is unimplemented");
-            }
```

Comment:
Shouldn't we keep these in since they are non-implemented parts of the discv5 spec?